content/power-on-failure-desktop.md

```diff
@@ -23,10 +23,11 @@ tableOfContents: true
 2. Confirm the outlet works by plugging in another device.
 3. Be sure that the system is on a flat surface
 4. Desktops have a button on the power supply located at the back of the computer. Toggle that button and make sure it is in the 'On' position though it may be labeled as a '1' to note that it is powered on, then attempt to turn the desktop on.
-5. If the power button lights up, the power supply is providing power to the machine. It is recommended to check the display connections next.
+5. Disconnect the power cable and switch the button on the power supply to off. Hold down the on/off button on the computer three times for 10 seconds each. This will drain capacitors and can potentially fix a failure to power on.
```

Comment:
I can agree to your first statement. I just like being as thorough as I can. I do not agree with your second statement at all. I've had too many cases of someone counting to 10 as quickly as they can which is often insufficient. I could meet you in the middle I suppose. It would take at least 10 seconds for some to count to 30 as quickly as they can. Further conversation amongst the team confirms that two separate times at least would be more advisable than just one time. We can say two times for 10 seconds each. Whether a system is out of box or not does not actually matter. This process can work for essentially any type of electronic device with a power supply attached to it whether it is new or old. As far as the why in your second statement. Three times for 10 seconds each was SOP in troubleshooting a failure to power up for the last 8 years for me. I would suggest that over anything else for the sake of being thorough.

Reply:
I disagree. There is no technical reason multiple times are needed.

Reply:
> It would take at least 10 seconds for some to count to 30 as quickly as they can. Further conversation amongst the team confirms that two separate times at least would be more advisable than just one time.

Why would two be more advisable than just one if it would take at least 10 seconds for someone to count to 30? This doesn't make sense.

Reply:
Furthermore, telling someone to hold it three times as long as you want them to because you assume they won't follow the instructions is not the way to go, because it adds nonsensical information for everyone who is paying attention and following the instructions. If you're concerned, a sub-bullet point note like `Be sure to hold the button for the entire 15 seconds; releasing too early can prevent the procedure from working.` would be more straightforward and transparent.

Reply:
For comparison, our CMOS clearing instructions in tech docs all just say 15 seconds, and we have not had a problem with users following those. The wording in tech docs is also more general than "capacitors" since other parts (such as the PSU coils) could also be carrying power.

> ...hold down the power button for at least 15 seconds to discharge any residual energy in the system.

Reply:
I've tweaked this in https://github.com/system76/docs/pull/1053/commits/eb4937eeca32948051d530db61ae62e159abcd41 in case capacitor draining is still valuable to the support docs.